translations/README.portuguese.md

```diff
@@ -0,0 +1,195 @@
+# Hasura GraphQL Engine
+
+[![Docs](https://img.shields.io/badge/docs-v1.0-brightgreen.svg?style=flat)](https://docs.hasura.io)
+[![CircleCI](https://circleci.com/gh/hasura/graphql-engine.svg?style=shield)](https://circleci.com/gh/hasura/graphql-engine)
+
+
+<a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://eepurl.com/dBUfJ5"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura GraphQL Engine é um servidor extremamente rápido que oferece a você criar **instantâneamente, em tempo real, APIs GraphQL baseadas no Postgres**, com [**webhook triggers**](event-triggers.md) em eventos do banco de dados, e [**remote schemas**](remote-schemas.md) para a regra de negócio.
+
+Hasura te ajuda a contruir aplicativos GraphQL suportados pelo Postgres ou migrar uma aplicação já existente, que utiliza Postgres, para GraphQL.
+
+Leia mais em [hasura.io](https://hasura.io) e na [documentação](https://docs.hasura.io).
+
+------------------
+
```

Comment:
This doesn't show up correctly:

![image](https://user-images.githubusercontent.com/5722022/67632563-4d332300-f8a5-11e9-9902-56418e6a5def.png)

Reply:
Resolved show up problem!